include/lsst/afw/geom/SpanSet.h

```diff
@@ -617,6 +620,23 @@ class SpanSet {
      */
     std::vector<std::shared_ptr<geom::SpanSet>> split() const;
 
+protected:
+    /* Returns the name used by the persistence layer to identify the SpanSet class
+     */
+    virtual std::string getPersistenceName() const;
+
+    /* Return a string corresponding to the python module that SpanSets lives in
+     */
+    virtual std::string getPythonModule() const;
```

Comment:
There's really no choice.  We have to store a string that tells us the module to `import` in order to unpersist the object, or we wouldn't be able to persist objects defined in plug-in packages.  The only way to avoid that would be to move all persistence down to Python.  That might be a good idea, but I don't think this issue is a big player in that cost/benefit analysis.